spring-cloud-gcp-samples/pom.xml

```diff
@@ -116,12 +117,20 @@
 	</profiles>
 
 	<build>
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<groupId>com.google.cloud.tools</groupId>
+					<artifactId>appengine-maven-plugin</artifactId>
+					<version>${app-engine-maven-plugin.version}</version>
+				</plugin>
+			</plugins>
+		</pluginManagement>
 		<plugins>
 			<plugin>
-				<artifactId>maven-deploy-plugin</artifactId>
-				<configuration>
-					<skip>true</skip>
```

Comment:
I believe this was necessary to skip publishing the module when the Spring Team runs `mvn deploy`. The default of the plugin is to not skip.

Reply:
I went looking for the `deploy` plugin up the chain of POMs - maybe I'm confused and it's just built-in? In that case, yes, I need to add this back in at the `spring-cloud-gcp-samples/pom.xml`, but I think then, it would cascade down and all the samples would _not_ need it, correct?

Reply:
It is built in, but I don't know about cascading.